features/admin.applications.v1/components/forms/inbound-oidc-form.tsx

```diff
@@ -1913,6 +1913,7 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
                                                     getAllowedList(metadata?.fapiMetadata?.tokenEndpointAuthMethod)
                                                     : getAllowedList(metadata?.tokenEndpointAuthMethod) }
                                                 readOnly={ readOnly }
+                                                data-testid={ `${ testId }-client-auth-method-dropdown` }
```

Comment:
Since this a new effort better to add `data-componentid` instead of `data-testid`

Reply:
+1. Let's use ` componentid` for newly written tests.

Reply:
fixed with 3b7e95bcb14d6e0edae4b48b13677e90896f5861